addons/l10n_din5008/i18n/nl.po

```diff
@@ -0,0 +1,281 @@
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 16.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-07-05 13:40+0000\n"
+"PO-Revision-Date: 2023-07-12 15:39+0200\n"
+"Last-Translator: Jort ten Donkelaar  <jort@aardug.eu>, 2023\n"
+"Language-Team: \n"
+"Language: nl\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#. module: l10n_din5008
+#: model_terms:ir.ui.view,arch_db:l10n_din5008.din5008_css
+msgid ""
+"&amp;.din_page {\n"
+"                        &amp;.header {\n"
+"                            .company_header {\n"
+"                                .name_container {\n"
+"                                    color:"
+msgstr ""
+
+#. module: l10n_din5008
+#: model_terms:ir.ui.view,arch_db:l10n_din5008.din5008_css
+msgid ""
+";\n"
+"                                    }\n"
+"                                }\n"
+"                            }\n"
+"                            h2 {\n"
+"                                color:"
+msgstr ""
+
+#. module: l10n_din5008
+#: model_terms:ir.ui.view,arch_db:l10n_din5008.din5008_css
+msgid ""
+";\n"
+"                                }\n"
+"                            }\n"
+"                        }\n"
+"                        &amp;.invoice_note {\n"
+"                            td {\n"
+"                                .address {\n"
+"                                    &gt; span {\n"
+"                                        color:"
+msgstr ""
+
+#. module: l10n_din5008
+#: model_terms:ir.ui.view,arch_db:l10n_din5008.din5008_css
+msgid ""
+";\n"
+"                            }\n"
+"                            .page {\n"
+"                                [name=invoice_line_table], [name=stock_move_table], .o_main_table {\n"
+"                                    th {\n"
+"                                        color:"
+msgstr ""
+
+#. module: l10n_din5008
+#: model_terms:ir.ui.view,arch_db:l10n_din5008.external_layout_din5008
+msgid "<span>|</span>"
+msgstr "<span>|</span>"
+
+#. module: l10n_din5008
+#: model:ir.model,name:l10n_din5008.model_account_analytic_line
+msgid "Analytic Account"
+msgstr "Kostenplaats"
+
+#. module: l10n_din5008
+#: model_terms:ir.ui.view,arch_db:l10n_din5008.external_layout_din5008
+msgid "BIC:"
+msgstr "BIC:"
+
+#. module: l10n_din5008
+#: model:ir.model.fields,field_description:l10n_din5008.field_base_document_layout__bank_ids
+msgid "Banks"
+msgstr "Banken"
+
+#. module: l10n_din5008
+#. odoo-python
+#: code:addons/l10n_din5008/models/account_move.py:0
+#, python-format
+msgid "Cancelled Invoice"
+msgstr "Geannuleerde factuur"
+
+#. module: l10n_din5008
+#: model:ir.model.fields,field_description:l10n_din5008.field_base_document_layout__city
+msgid "City"
+msgstr "Plaats"
+
+#. module: l10n_din5008
+#: model:ir.model,name:l10n_din5008.model_base_document_layout
+msgid "Company Document Layout"
+msgstr "Bedrijf document lay-out"
```

Comment:
Documentlay-out bedrijf?

Reply:
This, same with "creditfactuur", we're taken from existing translations in the account module. This one could be both but for the sake of consitency "Bedrijf document lay-out" I think is best. "Creditfactuur" should be correct as that's the most common translation dutch translations of "Credit Note" (within Odoo)